app/src/main/java/com/duckduckgo/app/browser/logindetection/FireproofDialogsEventHandler.kt

```diff
@@ -39,19 +38,21 @@ interface FireproofDialogsEventHandler {
     suspend fun onDisableLoginDetectionDialogShown()
     suspend fun onUserConfirmedDisableLoginDetectionDialog()
     suspend fun onUserDismissedDisableLoginDetectionDialog()
+    suspend fun onUserEnabledAutomaticFireproofLoginDialog(domain: String)
```

Comment:
I was a bit confused by the naming here, could we rename this to: `onUserEnabledAutomaticFireproofing()`?